vignettes/minimal-rstan-package.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "A working example for writing a package that depends on RStan"
+author: "Stefan Siegert"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Minimal RStan Package}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo=FALSE, eval=TRUE}
+knitr::opts_chunk$set(eval=FALSE)
+```
+
+## Creating a minimal package that uses Stan, step-by-step
+
+This vignette is a step-by-step guide to create a functioning R package that uses Stan.
+It is complementary to the [rstantools developer guidelines vignette](developer-guidelines.html) which you should read first.
+
+
+To start off, we use `rstan_package_skeleton` to initialise a bare-bones package directory.
+The name of our package will be `rstanlm`; it will fit a simple linear regression model using Stan.
+Under the hood, the function `rstan_package_skeleton` calls `utils::package.skeleton()` which won't allow creating a completely empty package directory, so we initialise it with a dummy variable which we will delete later on:
+
```

Comment:
I think you can call `rstan_package_skeleton("rstanlm")` without the dummy environment list. Does that result in an error for you?